src/bptl/work_units/zac/tasks.py

```diff
@@ -0,0 +1,70 @@
+import json
+
+from requests import Response
+
+from bptl.tasks.base import WorkUnit, check_variable
+from bptl.tasks.registry import register
+
+from .client import ZACClient
+from .serializers import ZacUsersDetailsSerializer
+
+
+@register
+class UserDetailsTask(WorkUnit):
+    """
+    Requests email and name data from usernames from the zac.
+
+    In the camunda process models accorderen/adviseren we have a list of usernames from
+    the zac. In order to send signalling emails, we will need to fetch
```

Comment:
```suggestion
    the zac. In order to send signaling emails, we will need to fetch
```

Reply:
British/American spellings are always tricky :-) (Mother is British, wife is American)